ValheimPlus/GameClasses/Monster.cs

```diff
@@ -0,0 +1,23 @@
+﻿using HarmonyLib;
+using ValheimPlus.Configurations;
+
+namespace ValheimPlus.GameClasses
+{
+    class Monster
+    {
+        /// <summary>
+        /// Altering time for loot appear
+        /// </summary>
+        [HarmonyPatch(typeof(Ragdoll), "Awake")]
```

Comment:
If the affected class is Ragdoll, the class name of the file should be Ragdoll.cs